jerry-main/main-unix.c

```diff
@@ -144,7 +144,7 @@ print_unhandled_exception (jerry_value_t error_value)
   }
   err_str_buf[err_str_size] = 0;
 
-  jerry_port_log (JERRY_LOG_LEVEL_ERROR, "Script Error: unhandled exception: %s\n", err_str_buf);
+  jerry_port_log (JERRY_LOG_LEVEL_ERROR, "Unhandled Script Error: %s\n", err_str_buf);
```

Comment:
From the list of alternatives you've suggested, I'd go for "Script Error: ", so even "Unhandled" might be removed.